feature_engine/selection/base_selector.py

```diff
@@ -25,7 +25,9 @@ def get_feature_importances(estimator):
         else:
             importances = np.linalg.norm(coef_, axis=0, ord=len(estimator.coef_))
 
-    return list(importances)
```

Comment:
the issue - untracked by mypy up to this point - was caused by _importances_ potentially being None (and list(None) isn't allowed)